f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -1488,6 +1488,9 @@ def __init__(self, driver, **kwargs):
         elif subtype == 'TCP':
             monitor_type = resource_helper.ResourceType.tcp_monitor
             self._resource = 'tcp_monitor'
+        elif subtype == 'UDP':
+            monitor_type = resource_helper.ResourceType.udp_monitor
+            self._resource = 'udp_monitor'
```

Comment:
If we need to add protocol UDP in pool as well ?

Reply:
seems not